Monika After Story/game/zz_reactions.rpy

```diff
@@ -2855,3 +2855,49 @@ label mas_reaction_gift_clothes_velius94_dress_whitenavyblue:
     if giftname is not None:
         $ store.mas_filereacts.delete_file(giftname)
     return
+
+init 5 python:
+    if not renpy.seen_label("mas_reaction_gift_carddeck"):
```

Comment:
We should probably do a property for this at some point in the `addReaction` function, which would allow us to only gift something once.